rpi/page.py

```diff
@@ -127,8 +127,8 @@ def __init__(self, main_view, *args, **kwargs):
 		self.fingerprint_enrollment_btn.pack(pady=10)
 		self.register_user_btn.pack(pady=10)
 		
-		def attempt_register(self, username, password):
-			credentials = {"username": username, "password": password}
+		def attempt_register(self, username, password, fingerprintId):
+			credentials = {"username": username, "password": password, "fingerprintId": fingerprintId, "sensorId": 1}
```

Comment:
Not technically **credentials**. Let's rename it to something like **payload** or **data**.